containers/hawkular-metrics-index-truncator/src/main/resources/logback.xml

```diff
@@ -0,0 +1,35 @@
+<!--
+
+    Copyright 2014-2019 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<configuration>
```

Comment:
I'm still not seeing any logs in stdout of the process except this
```
log4j:WARN No appenders could be found for logger (com.openshift.internal.restclient.ApiTypeMapper).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
```

unless I add a `log4j.properties` which fixes it